apps/agent/app/src/components/AgentSvg.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
```

Comment:
Let's optimize this into a single asset, perhaps @Evalir you can help @ECWireless do this?

Reply:
I merely ran the SVG through [svg-to-react-cli](https://www.npmjs.com/package/svg-to-react-cli) originally, but I believe it's optimized in my latest commit (?) @Evalir definitely wouldn't mind help if this is wrong though!

Reply:
Yup, it is better now. :)